### PR TITLE
Add Ts.ED demo

### PR DESCRIPTION
**Pull request by pbn4**

### CODE REVIEW

File: services/listings/ormconfig.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+module.exports = {
+  name: "default",
+  type: "postgres",
+  database: "bloom",
+  cli: {
+    migrationsDir: `services/listings/src/migrations`,
+  },
+  entities: [`${__dirname}/src/entity/*{.ts,.js}`],
+  migrations: [`${__dirname}/src//migrations/*{.ts,.js}`],
```
Comment (software-project) [inline services/listings/ormconfig.js:10]: Single backslash here?

File: services/listings/package.json [hunk 2]
```diff
@@ -14,24 +14,58 @@
...
     "jest": "^25.2.3",
     "supertest": "^4.0.2",
-    "ts-node-dev": "^1.0.0-pre.44"
+    "ts-node-dev": "^1.0.0-pre.44",
+    "tsconfig-paths": "^3.9.0"
   },
   "dependencies": {
     "@bloom-housing/core": "^0.1.0",
     "@koa/cors": "^3.0.0",
+    "@storybook/react": "^5.3.14",
```
Comment (software-project) [inline services/listings/package.json:26]: We need storybook for Ts.ED?

Comment (pbn4) [reply services/listings/package.json]: Good catch. I am not sure how it got there in the first place.

**Comment by bk3c** [comment]
Closing this in favor of moving forward with NestJS as discussed in #263. Thanks for looking into it, @pbn4.

**Pull request closed without merging**